bin/docker-command.bash

```diff
@@ -55,17 +55,15 @@ if [ ! -f /provisioned ] || [ "$FORCE_PROVISION" = "on" ]; then
     fi
 
     if [ "$DISABLE_INITIAL_LOADDATA" != "on" ]; then
-
-        python manage.py loaddata app/fixtures/users.json
-        python manage.py loaddata app/fixtures/economy.json
-        python manage.py loaddata app/fixtures/profiles.json
-        python manage.py loaddata app/fixtures/kudos.json
-        python manage.py loaddata app/fixtures/grants.json
-        python manage.py loaddata app/fixtures/dashboard.json
-        python manage.py loaddata app/fixtures/avatar.json
-        python manage.py loaddata app/fixtures/marketing.json
-        python manage.py loaddata app/fixtures/profiles.json
-        python manage.py loaddata app/fixtures/grants.json
+        python3 manage.py loaddata app/app/fixtures/users.json
+        python3 manage.py loaddata app/app/fixtures/economy.json
+        python3 manage.py loaddata app/app/fixtures/profiles.json
+        python3 manage.py loaddata app/app/fixtures/kudos.json
+        python3 manage.py loaddata app/app/fixtures/grants.json
+        python3 manage.py loaddata app/app/fixtures/dashboard.json
+        python3 manage.py loaddata app/app/fixtures/avatar.json
+        python3 manage.py loaddata app/app/fixtures/marketing.json
+        python3 manage.py loaddata app/app/fixtures/grants.json
```

Comment:
- grants is duplicated
- only users, economy, profiles and marketing seem to work